cdc/entry/mounter.go

```diff
@@ -332,11 +474,29 @@ func (m *mounter) mountRowKVEntry(tableInfo *model.TableInfo, row *rowKVEntry, d
 	if row.PreRowExist {
 		// FIXME(leoppro): using pre table info to mounter pre column datum
 		// the pre column and current column in one event may using different table info
-		preCols, preRawCols, err = datum2Column(tableInfo, row.PreRow, m.enableOldValue)
+		preCols, preRawCols, columnIDs, err = datum2Column(tableInfo, row.PreRow, m.enableOldValue)
 		if err != nil {
 			return nil, rawRow, errors.Trace(err)
 		}
 
+		preChecksum, matched, err = m.verifyChecksum(columnIDs, preRawCols, true)
+		if err != nil {
+			return nil, rawRow, errors.Trace(err)
+		}
+
+		if !matched {
+			log.Error("pre columns checksum mismatch",
```

Comment:
What does `pre` mean?

Reply:
I mean the previous columns, I will refine the log.